ui/src/app/components/admin/audit-card-list-view.tsx

```diff
@@ -21,26 +22,61 @@ const HideableCell = (props: {content: string}) => {
       }}>{content}</div>;
 };
 
+const HideableLinkCell = (props: {url: string, content: string}) => {
+  const {content, url} = props;
+  return <a
+      style={{backgroundColor: content ? 'white' : '#f0f3f5',
+        border: '1px solid',
+        boxSizing: 'border-box'}}
+        href={url}>{content}</a>;
+};
+
 const styles = reactStyles({
   propertyCell: {
     fontWeight: 600,
     border: '1px solid'
   }
 });
 
-const PropertyChangeListEntry = (props: {targetProperty?: string, previousValue?: string, newValue?: string}) => {
-  const {targetProperty, previousValue, newValue} = props;
-  return <React.Fragment>
+const isWorkspaceNamespace = (targetType?: string, targetProperty?: string) => {
+  return targetType === 'WORKSPACE' && targetProperty === 'namespace';
+};
+
+const PossibleLinkCell = (props: {targetType?: string, targetProperty?: string, value?: string}) => {
+  const {targetType, targetProperty, value} = props;
+  if (isWorkspaceNamespace(targetType, targetProperty)) {
```

Comment:
My style is generally lots of tiny named functions. I don't really like the ternary operator in any language, or the if statement for  that matter. It usually means I've missed an abstraction somewhere. This bit  is pretty hacky though.

Reply:
I like tiny functions as well. In JS my personal preference is to in-line them unless I am using them in more than one place.

Agreed on the conditionals as well. The reason I prefer ternary is because it is a "this or that" operator. With `if` statements it makes it easier for a future developer to add additional conditions and nested conditionals rather than finding that better abstraction. The change from a ternary to a more complex conditional may stick out a bit more and raise a flag for someone - but that may just be me.